tests/integration/runners/test_fileserver.py

```diff
@@ -202,15 +202,31 @@ def test_update(self):
         fileserver.update
         """
```

Comment:
This test needs to be migrated to pytests. We are no longer accepting non-pytest tests.